sdk/utils/network.go

```diff
@@ -0,0 +1,68 @@
+package utils
```

Comment:
nitpick: maybe we could name this package `network`, with methods called `Setup` and `SetupWaitAll`, then we avoid having a generic name for a package.

Reply:
Agree. If we do that, we should move all network helpers here (e.g., `sync.WaitNetworkInitialized`).

Reply:
Sounds good, I will do that.